### PR TITLE
Fix missing pname in environment parsing

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

File: imjoy/imjoyPluginEngine.py [hunk 1]
```diff
@@ -1654,8 +1657,9 @@ def launch_plugin(
             "Failed to setup plugin virual environment or its requirements: " + str(e),
             type="error",
         )
+        error = e
```
Comment (MartinHjelmare) [inline imjoy/imjoyPluginEngine.py]: It looks weird that we don't take further action here, upon error. That's why I added this error variable, which we check below and abort if it's truthy. Not sure if that's the correct way of doing things.

File: imjoy/imjoyPluginEngine.py [hunk 2]
```diff
@@ -634,7 +634,7 @@ def runCmd(
     return output
 
 
-def parseEnv(env, work_dir):
+def parseEnv(env, work_dir, pname):
```
Comment (oeway) [inline imjoy/imjoyPluginEngine.py]: maybe change `pname` to something like `default_env_name`, and when call this function use `default_env_name= pname + '-' +tag if tag != '' else pname`?

Comment (MartinHjelmare) [reply imjoy/imjoyPluginEngine.py]: Ok.

File: imjoy/imjoyPluginEngine.py (hunk 1 above)
Comment (oeway) [reply imjoy/imjoyPluginEngine.py]: I think you can do `abort.set()` instead of introducing `error`.

Comment (MartinHjelmare) [reply imjoy/imjoyPluginEngine.py]: Ok.

**Pull request merged**